src/commands/emulators-start.ts

```diff
@@ -18,6 +22,46 @@ module.exports = new Command("emulators:start")
     }
 
     utils.logLabeledSuccess("emulators", "All emulators started, it is now safe to connect.");
+    const table = new Table({
+      head: ["Emulator", "Host:Port", "View in Browser"],
+      style: {
+        head: ["yellow"],
+      },
+    });
+
+    const guiInfo = EmulatorRegistry.getInfo(Emulators.GUI);
+    table.push(
+      ...controller
+        .filterEmulatorTargets(options)
+        .map((emulator) => {
+          const info = EmulatorRegistry.getInfo(emulator);
+          const emulatorName = emulator.slice(0, 1).toUpperCase() + emulator.slice(1);
+          const isSupportedByGUI = EMULATORS_SUPPORTED_BY_GUI.includes(emulator);
+
+          if (!info) {
+            return [emulatorName, "Failed to initialize (see above)", ""];
+          }
+
+          return [
+            emulatorName,
+            `${info?.host}:${info?.port}`,
+            isSupportedByGUI ? `http://${guiInfo?.host}:${guiInfo?.port}/${emulator}` : "",
```

Comment:
This will totally give you `http://undefined:undefined/` (or null, or empty string) if GUI is not running. Please replace with:
```suggestion
            isSupportedByGUI && guiInfo ? `http://${guiInfo.host}:${guiInfo.port}/${emulator}` : "",
```